x-pack/plugins/apm/common/fleet.ts

```diff
@@ -8,7 +8,7 @@ import semverParse from 'semver/functions/parse';
 
 export const POLICY_ELASTIC_AGENT_ON_CLOUD = 'policy-elastic-agent-on-cloud';
 
-export const SUPPORTED_APM_PACKAGE_VERSION = '7.16.0';
+export const SUPPORTED_APM_PACKAGE_VERSION = '8.0.0-dev4'; // TODO update to just '8.0.0' once published
```

Comment:
once 8.0.0 is published in staging or snapshot deployments of the package-registry, this needs to be updated.